buildlib/pr/ucx-py/ucx-py-test.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+  name: ucx-py test
+  demands: []
+
+jobs:
+  - job: ${{ parameters.name }}
+
+    pool:
+      name: MLNX
+      demands: ${{ parameters.demands }}
+
+    steps:
+      - checkout: self
+        fetchDepth: 100
+        clean: true
+        displayName: Checkout
+      - bash: |
+          set -x
+          source buildlib/az-helpers.sh
+          az_init_modules
+          try_load_cuda_env
+          set -eE
+          ./autogen.sh
+          ./contrib/configure-devel --prefix=$(Build.Repository.LocalPath)/install \
+            --enable-gtest=no --with-cuda=$have_cuda
+          make -j`nproc`
+          make install
+        displayName: Build UCX
+      - bash: |
+          set -x
+          source buildlib/az-helpers.sh
+          az_init_modules
+          try_load_cuda_env
+          set -eE
+
+          ifaces=`get_rdma_interfaces`
+          if [ -z "$ifaces" ]; then
+              azure_log_warning "No active RDMA interfaces on machine"
```

Comment:
why require RDMA?

Reply:
Also from jucx. Will `rc`/`RDMACM` tests work without it?

Reply:
I see. so let's not skip ALL tests if no rdma. Just skip "tls=rc*"

Reply:
Done in https://github.com/openucx/ucx/pull/7412/commits/ecca05be22e6ea6888ddeba07c74b413568207d4 .